benchmark/src/main/scala/spire/benchmark/ScalaVsSpire.scala

```diff
@@ -235,7 +235,8 @@ class ScalaVsSpireBenchmarks {
   }
 
   @tailrec final def gcdSpire[@sp(Int) A](a: A, b: A)(implicit ev1: EuclideanRing[A], ev2: Eq[A]): A = {
-    import spire.implicits._
+    import spire.syntax.euclideanRing._
```

Comment:
This change was needed as there is `b` function in implicits as indicated here:
https://github.com/lampepfl/dotty/issues/13602

It can be fixed renaming the `b` param

Reply:
Yup I caught that one.